<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>564</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QGroupBox" name="groupBox_drone_infos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>191</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Informations</string>
    </property>
    <widget class="QLabel" name="label_speed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The maximum speed allowed by the drone</string>
     </property>
     <property name="text">
      <string>Max. speed</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_flight_time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The flight time session in seconds</string>
     </property>
     <property name="text">
      <string>Flight time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_height">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The actual height in centimers of the drone</string>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_temperature">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The drone temperature in degrees celcius</string>
     </property>
     <property name="text">
      <string>Temperature</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_imu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Inertial Measurement Unit: Yaw, Pitch, Roll</string>
     </property>
     <property name="text">
      <string>I.M.U</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pressure">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The atmospheric pressure in millibars</string>
     </property>
     <property name="text">
      <string>Pressure</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_acceleration">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The acceleration vector of the drone: x, y, z</string>
     </property>
     <property name="text">
      <string>Acceleration</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tof">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Time Of Flight: Height in centimers relative from the take off point</string>
     </property>
     <property name="text">
      <string>T.O.F</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_wifi">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The Wifi Signal Noise Ratio</string>
     </property>
     <property name="text">
      <string>Wifi SNR</string>
     </property>
    </widget>
    <widget class="QLabel" name="drone_speed">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0 cm /s</string>
     </property>
    </widget>
    <widget class="QLabel" name="drone_flight_time">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0 s</string>
     </property>
    </widget>
    <widget class="QLabel" name="drone_height">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0 cm</string>
     </property>
    </widget>
    <widget class="QLabel" name="drone_temperature">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0Â°C</string>
     </property>
    </widget>
    <widget class="QLabel" name="drone_imu">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0, 0, 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="drone_pressure">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0 m</string>
     </property>
    </widget>
    <widget class="QLabel" name="drone_acceleration">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0, 0, 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="drone_tof">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0 cm</string>
     </property>
    </widget>
    <widget class="QLabel" name="drone_wifi">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>191</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QPushButton" name="button_takeoff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Take off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_emergency">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: red;
}</string>
     </property>
     <property name="text">
      <string>EMERGENCY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_start_video">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start video</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_connexion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Connexion</string>
    </property>
    <widget class="QLineEdit" name="input_ip">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.10.1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_connect">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CONNECT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_port">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>PORT</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="input_port">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>8889</number>
     </property>
    </widget>
   </widget>
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>581</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
